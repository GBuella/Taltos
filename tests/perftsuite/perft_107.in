setboard 7k/8/1p6/8/8/P7/8/7K w - - 0 1
echo 7k/8/1p6/8/8/P7/8/7K w - - 0 1
qperft 1
qperft 2
qperft 3
qperft 4
qperft 5
qperft 6
