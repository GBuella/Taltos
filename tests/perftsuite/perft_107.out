7k/8/1p6/8/8/P7/8/7K w - - 0 1
4
16
101
637
4354
29679
